tests/e2e-leg-9/replicated-upgrade-sanity/45-verify-sandbox-access.yaml

```diff
@@ -1,20 +0,0 @@
-# (c) Copyright [2021-2024] Open Text.
-# Licensed under the Apache License, Version 2.0 (the "License");
-# You may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-# Verifies access to the external service through port 5433.  It does this
-# by connecting using vsql
-
-apiVersion: kuttl.dev/v1beta1
-kind: TestStep
-commands:
-  - command: bash -c "kustomize build verify-sandbox-connection/overlay | kubectl -n $NAMESPACE apply -f - "
```

Comment:
Why do we remove this connection verification?

Reply:
After we promote sandbox to main, we can't connect to sandbox anymore.

Reply:
I think when I revert this step back, I can still connect to new main cluster.